.gitignore

```diff
@@ -1,3 +1,4 @@
 target
 Cargo.lock
 .DS_Store
+.idea
```

Comment:
Missing a backline after this line.

Also I'm curious: what's this file?

Reply:
Just pushed, thanks.

.idea is an autogenerated config directory for JetBrains IDEs